### PR TITLE
week1-day5 added no9 and pairs files

### PR DESCRIPTION
**Pull request by aaronshivers**

### CODE REVIEW

File: pairs.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return string.charAt(string.length - 1);
+  }
+
+  static getPairMap(array: string[]) {
+    this.createPairMap(array);
+    return this.pairMap;
+  }
+}
+
+// console.log(Pairs.getPairMap([ 'code', 'bug' ])); // {"b": "g", "c": "e"}
```
Comment (ramoosdclass) [inline pairs.ts]: remove commented code

File: pairs.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    Pairs.getLastCharacter = function (string) {
+        return string.charAt(string.length - 1);
+    };
+    Pairs.getPairMap = function (array) {
+        this.createPairMap(array);
+        return this.pairMap;
+    };
+    Pairs.pairMap = {};
+    return Pairs;
+}());
```
Comment (ramoosdclass) [inline pairs.js:22]: remove commented code

**Review by ramoosdclass — CHANGES_REQUESTED** [review]

**Comment by ramoosdclass** [comment]
no9.ts is empty please finish that in ts as well

File: pairs.js (hunk 2 above)
Comment (ramoosdclass) [reply pairs.js:22]: https://github.com/aaronshivers/angular-path/pull/9/commits/b05fd9d82bc3f998169d92516387b5f34db94bfe

**Review by ramoosdclass — APPROVED** [review]

File: pairs.ts (hunk 1 above)
Comment (aaronshivers) [reply pairs.ts]: https://github.com/aaronshivers/angular-path/pull/9/commits/0a504b11e495f8ff41deb3c924c7ab4602e54a88

**Pull request merged**